### ISSUE TITLE
Sound suggestion

### ISSUE THREAD

**Issue by cuccarese** [GaijinEntertainment/fmod_studio_warthunder_for_modders#292]
I was told I could post suggestions here, so this is what I thought. 
When switching certain rank thresholds, there should be a music change (I don't know the correct term; I'm new to this). For example, the music should be changed from rank four to six to a more 60s --> 2000s theme. This shows the user has transitioned from WW2 ranks to Cold War ranks. This could be refined further by going from CW to Modern, but a baseline change would be better. 

Rank 1 - scratchy gramophone music
Ranks 2 - 4 WW2 radio music
Ranks 5 - 6 60's to 80's themed music (i.e., synth, light rock beats, disco-themed)
Ranks 7 & 8 90's to 2000's themed music (i.e., pop beats, fast rap tracks)

Implementing this framework could open up exciting possibilities for modders. They could create unique sound mods, adding a new layer of immersion for enthusiasts. This could potentially lead to increased downloads, benefiting both the modders and the game. For the base game, reviewers may be more inclined to praise the level of detail, which can be a compelling selling point.

**Comment by pixtolero** [comment]
good suggestion

**Comment by Uchuujin01** [comment]
This is a really nice suggestion, we'll discuss it, thank you. However, writing of the new music demands significant time. What you suggest is literally means writing of 4 whole game music arrangements instead of current 2.

**Comment by cuccarese** [comment]
@Uchuujin01, I agree that it would take time. That being said, could you develop the framework and let the modders test it out (internally among themselves or externally among the people who use it)? If it's received well, you could push a version out for everyone to use with music created by the team or buy existing music rights. Even better, it could be used as a community event where people compose their own music and send it in as a competition, and the winner receives like a $70 pack, and their music added, and the runner-ups get like 5k GE with theirs added as well.
